app/controllers/v1/audits_controller.rb

```diff
@@ -17,16 +17,11 @@ def show
 
     def create
       collection_policy.new(current_user).authorize! :new?
-
-      packages = Package.stored
-      audit = Audit.new(user: current_user, packages: packages.count)
+      audit = Audit.new(user: current_user)
 
       resource_policy.new(current_user, audit).authorize! :save?
-      audit.save
 
-      packages.each do |package|
-        AuditFixityCheckJob.perform_later(package: package, user: current_user, audit: audit)
-      end
+      AuditManager.start_audit(user: current_user, audit: audit)
```

Comment:
Because the total logic is brief and each line is simple, I would remove the blank lines between them and put one before the response. Then the business part is grouped together, and the web part finishes the method alone.